.rustfmt.toml

```diff
@@ -0,0 +1,24 @@
+# stable options
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
+merge_derives = false
+newline_style = "Auto"
+remove_nested_parens = true
+reorder_imports = true
+reorder_modules = true
+tab_spaces = 4
+use_field_init_shorthand = true
+use_try_shorthand = true
+edition = "2018"
+
+# unstable options (for one-off fixes and CI)
+merge_imports = true
+wrap_comments = true
+comment_width = 80
```

Comment:
`80` is the default.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#comment_width

Reply:
Any reason not to have it at 100 to match the code width? If people find 80 more readable, that's fine with me, but I like to save the vertical space that 100 provides.

Reply:
Ah, good call.  I spaced out on that.  No reason - I will change to 100.